app/components/forms/orders/order-form.js

```diff
@@ -497,7 +497,10 @@ export default Component.extend(FormMixin, {
   },
 
   allFields: computed('fields', function() {
-    return groupBy(this.fields.toArray(), field => field.get('form'));
+    const requiredFixed = this.fields.toArray().slice(0, 3);
+    const customFields =  orderBy(this.fields.toArray().slice(3), ['isComplex', 'name'], ['asc', 'asc']);
```

Comment:
by default all values are sorted in ascending order

Reply:
no they are not in a way in attendee form.

Reply:
I mean orderBy funtion by default I think sort all values in ascending order so no need to pass ```['asc', 'asc']```